hadoop-project/pom.xml

```diff
@@ -2747,6 +2747,16 @@
         </dependencies>
       </dependencyManagement>
     </profile>
+    <profile>
+      <id>jdk-17</id>
```

Comment:
can we follow the [apache/maven-apache-parent](https://github.com/apache/maven-apache-parent/blob/2591ca2d83b71d18a621d01621f078f25cfd456e/pom.xml#L501) to name it `jdk17+`

Reply:
Done.